### ISSUE TITLE
Doubts about Websockets implementation 

### ISSUE THREAD

**Issue by Marlysson** [vibora-io/vibora#24]
I've seen the websockets folder and ```obj.py``` file's implementation .. And there is a lot of mount implementation handling with bits ( low level ) creating masks, and more things ...

Do is your effort implement websockets whole from scratch ? Handle bit-to-bit of the request.....

**Comment by frnkvieira** [comment]
Yes. Vibora WebSockets is a from-scratch implementation. The one is this repository is just a prototype, hopefully, I'll push the complete version until the end of the week.
